src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -24,6 +23,21 @@ class FileMetadataService(fileMetadataRepository: FileMetadataRepository,
 
   val loggingUtils: LoggingUtils = LoggingUtils(Logger("FileMetadataService"))
 
+  def getPropertyType(propertyType: Option[String]): PropertyType = propertyType match {
```

Comment:
I would be tempted to move these functions out into their own service: `CustomMetadataPropertiesService` just to make it clear what they are, and distinguish them from the actual "metadata" for files.